lua/lib/util.lua

```diff
@@ -46,6 +46,26 @@ util.file_exists = function(name)
   end
 end
 
+--- query file size.
+-- @tparam string name filepath
+-- @treturn number filesize in mb
+util.file_size = function(path)
+  if path ~= nil then
+    local f = io.open(path,"r")
+    if f~=nil then
+      local c = f:seek()      -- get current position
+      local s = f:seek("end") -- get file size
+      f:seek("set", c)        -- restore position
+      io.close(f)
+      return s/1000000
```

Comment:
it might make sense to return bytes and let the script author choose the units to use, sometimes MB is 1048576 bytes too :)